pkg/p2p/metrics.go

```diff
@@ -19,6 +19,42 @@ import (
 )
 
 var (
+	serverStreamCount = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
the grafana template will added later after the code is stable ?

Reply:
Yes. Before releasing the peer-to-peer scheduling, Grafana panels will be added.